.github/workflows/checks.yaml

```diff
@@ -61,7 +61,13 @@ jobs:
         run: make deploy-ci
 
   e2e:
-    name: Istio End-to-End Test
+    name: Istio End-to-End Test (${{ matrix.variant }})
+    strategy:
+      matrix:
+        include:
+          - variant: main
+          - variant: tag
```

Comment:
Can you add a comment describing what this means ?

Reply:
Will do!